client/src/components/Reviews_Ratings/Ratings_Reviews.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import API_KEY from '../../../../config.js';
+import reviews_data from './sample_review_data.js';
+import ReviewList from './ReviewList.jsx'
+
+const Ratings_Reviews = () => {
+  const [reviews, getAllReviews] = useState([]);
+
+  const axiosGetAllReviews = () => {
+    axios.get(`https://app-hrsei-api.herokuapp.com/api/fec2/hratx/reviews`, {
+      headers: {
+        'Authorization': API_KEY
+      },
+      params: {
+        'product_id': 21112
+      }
```

Comment:
Is this just test data? If so, we might consider changing it before merging it with the production branch.